autogluon/task/text_prediction/models/basic_v1.py

```diff
@@ -710,7 +710,11 @@ def train(self, train_data, tuning_data, resource,
             raise NotImplementedError
         scheduler.run()
         scheduler.join_jobs()
-        self._logger.info('Best_config={}'.format(scheduler.get_best_config()))
+        if len(scheduler.config_history) == 0:
+            raise RuntimeError('No job has been completed! You may try to increase '
```

Comment:
Could be more clear if this instead read: "No training job has been completed"

Reply:
Improved the error message.